libs/src/libs.js

```diff
@@ -4,34 +4,36 @@ const { EthWeb3Manager } = require('./services/ethWeb3Manager')
 const { SolanaAudiusData } = require('./services/solanaAudiusData/index')
 const { Web3Manager } = require('./services/web3Manager')
 const { EthContracts } = require('./services/ethContracts')
-const { SolanaWeb3Manager } = require('./services/solana')
+const {
+  SolanaWeb3Manager,
+  SolanaUtils,
+  RewardsAttester
+} = require('./services/solana')
 const { AudiusContracts } = require('./services/dataContracts')
 const { IdentityService } = require('./services/identity')
 const { Comstock } = require('./services/comstock')
 const { Hedgehog } = require('./services/hedgehog')
 const { CreatorNode } = require('./services/creatorNode')
 const { DiscoveryProvider } = require('./services/discoveryProvider')
-const Wormhole = require('./services/wormhole')
+const { Wormhole } = require('./services/wormhole')
 const { AudiusABIDecoder } = require('./services/ABIDecoder')
 const { SchemaValidator } = require('./services/schemaValidator')
 const { UserStateManager } = require('./userStateManager')
 const SanityChecks = require('./sanityChecks')
 const { Utils, Captcha } = require('./utils')
 const { ServiceProvider } = require('./api/ServiceProvider')
 
-const Account = require('./api/account')
-const User = require('./api/user')
-const Track = require('./api/track')
-const Playlist = require('./api/playlist')
-const File = require('./api/file')
-const Rewards = require('./api/rewards')
+const { Account } = require('./api/Account')
+const { Users } = require('./api/Users')
```

Comment:
I was gonna say keep `Users` singular to match the convention but then I saw `Rewards` and `Reactions` :(